testing/test_multicall.py

```diff
@@ -197,3 +197,40 @@ def m2():
     with pytest.raises(exc):
         MC([m2, m1], {}).execute()
     assert l == ["m1 init", "m1 finish"]
+
+
+@hookimpl(hookwrapper=True)
+def m1(arg1, arg2, arg3):
+    yield
+
+
+@hookimpl
+def m2(arg1, arg2, arg3):
+    return arg1, arg2, arg3
+
+
+@hookimpl(hookwrapper=True)
+def w1(arg1, arg2, arg3):
+    yield
+
+
+@hookimpl(hookwrapper=True)
+def w2(arg1, arg2, arg3):
+    yield
+
+
+def inner_exec(methods):
+    return MC(methods, {'arg1': 1, 'arg2': 2, 'arg3': 3}).execute()
+
+@pytest.mark.benchmark
+def test_hookimpls_speed(benchmark):
+    benchmark(inner_exec, [m1, m2])
+
+
+@pytest.mark.benchmark
+def test_hookwrapper_speed(benchmark):
+    benchmark(inner_exec, [w1, w2])
```

Comment:
@ionelmc I didn't even think about the effects of the CI server.. very good point.

> or you could compare against a baseline (eg, you have a test that only does 10 function calls to establish a baseline to compare against - if there's a perf regression you will get slower than the baseline.

Can you elaborate on this? Are you suggesting this approach will work in CI?

I think I'll still include a baseline from the machine I develop on at least as a reference.

> We can talk about instruction counters I guess (since those shouldn't care about preemption right?), what would be useful?

Yes this would be super useful. Can I pull this from the profiling data; I'm guessing it can't be used with `--benchmark-compare-fail`?